companies/svit.json

```diff
@@ -0,0 +1,16 @@
+{
+    "slug": "svit",
+    "name": "SVIT Schweiz",
+    "address": "SVIT Schweiz\nGiessereistrasse 18\n8005 Zürich\nSchweiz",
+    "phone": "+41 44 434 78 88",
+    "fax": "41 44 434 78 99",
+    "email": "info@svit.ch",
+    "web": "https://www.svit.ch",
+    "sources": [
+        "https://www.svit.ch/de/datenschutz",
+        "https://www.svit.ch/de/impressum-svit-schweiz",
```

Comment:
You need to remove the comma at the end here, otherwise this is technically not a valid JSON and our automated tools will complain.